src/applications/vaos/components/layout/DetailPageLayout.jsx

```diff
@@ -202,31 +212,176 @@ function CancelButton({ appointment }) {
   return null;
 }
 
+export function TravelReimbursement({
+  appointmentDate,
+  claimData,
+  daysRemainingToFileClaim,
+  isPastAppointment,
+}) {
+  if (!isPastAppointment) return null;
+  if (!claimData) return null;
+
+  let body = null;
+
+  if (
+    claimData.message === TRAVEL_CLAIM_MESSAGES.noClaim &&
+    daysRemainingToFileClaim > 0
+  ) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          Days left to file: {daysRemainingToFileClaim}
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="file-claim-link"
+            className="vads-u-margin-y--0p5"
+            href={`/appointments/claims/?date=${appointmentDate}`}
+            text="File a travel reimbursement claim"
+          />
+        </p>
+      </>
+    );
+  }
+  if (
+    claimData.message === TRAVEL_CLAIM_MESSAGES.noClaim &&
+    daysRemainingToFileClaim < 1
+  ) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          Days left to file: {daysRemainingToFileClaim}
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          You didn’t file a claim for this appointment. You can only file for
+          reimbursement within 30 days of the appointment.
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="how-to-file-claim-link"
+            className="vads-u-margin-y--0p5"
+            href="https://www.va.gov/resources/how-to-file-a-va-travel-reimbursement-claim-online/"
+            text="Learn more about travel reimbursement"
+          />
+        </p>
+      </>
+    );
+  }
+  if (claimData.id) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          You've already filed a claim for this facility and date.
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="view-claim-link"
+            href={`/appointments/claims/${claimData.id}`}
```

Comment:
I'm assuming this end point will be built in the future?

Reply:
I got an answer in this [thread](https://dsva.slack.com/archives/C05Q9HH47PS/p1730830808403269) `my-health/travel-claim-status/[claim.id]`